composer.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "shogun/frontend-checkout",
+    "description": "Adobe Commerce Cloud extension to checkout orders from Shogun storefronts.",
```

Comment:
I'd suggest a more generic description, just mentioning it's integrating Magento with Frontend